compact/far_forward/vacuum.xml

```diff
@@ -1,5 +1,5 @@
 <!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
-<!-- Copyright (C) 2022 Whitney Armstrong, Alex Jentsch -->
+<!-- Copyright (C) 2023 Alex Jentsch -->
```

Comment:
Keep previous years and names (unless you think the previous names were a mistake or this is a full rewrite).

Reply:
It was essentially a full re-write in both cases - and for the .cpp file, I had originally written it from scratch, but copy and pasted the skeleton from a template during the ATHENA times. I just want to make sure the relevant person is contacted in the event of a question.